tests/manual-test-cases/Group5-Functional-Tests/5-3-Enhanced-Linked-Mode.robot

```diff
@@ -0,0 +1,136 @@
+    Should Contain  ${out}  OK
+    ${out}=  Run  govc dvs.portgroup.add -nports 12 -dc=ha-datacenter -dvs=test-ds bridge
+    Should Contain  ${out}  OK
+
+    Log To Console  Add all the hosts to the distributed switch
+    ${out}=  Run  govc dvs.add -dvs=test-ds -pnic=vmnic1 /ha-datacenter/host/cls
+    Should Contain  ${out}  OK
+
+    Log To Console  Enable DRS on the cluster
+    ${out}=  Run  govc cluster.change -drs-enabled /ha-datacenter/host/cls
+    Should Be Empty  ${out}
+    
+    # Second VC cluster
+    Set Environment Variable  GOVC_URL  ${vc2-ip}
+    Log To Console  Create a datacenter on the VC
+    ${out}=  Run  govc datacenter.create ha-datacenter
+    Should Be Empty  ${out}
+    
+    Log To Console  Create a cluster on the VC
+    ${out}=  Run  govc cluster.create cls
+    Should Be Empty  ${out}
+
+    Log To Console  Add ESX host to the VC
+    ${out}=  Run  govc cluster.add -hostname=${esx4-ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true
+    Should Contain  ${out}  OK
+    ${out}=  Run  govc cluster.add -hostname=${esx5-ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true
+    Should Contain  ${out}  OK
+    ${out}=  Run  govc cluster.add -hostname=${esx6-ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true
+    Should Contain  ${out}  OK
+
+    Log To Console  Create a distributed switch
+    ${out}=  Run  govc dvs.create -dc=ha-datacenter test-ds
+    Should Contain  ${out}  OK
+
+    Log To Console  Create three new distributed switch port groups for management and vm network traffic
+    ${out}=  Run  govc dvs.portgroup.add -nports 12 -dc=ha-datacenter -dvs=test-ds management
+    Should Contain  ${out}  OK
+    ${out}=  Run  govc dvs.portgroup.add -nports 12 -dc=ha-datacenter -dvs=test-ds vm-network
+    Should Contain  ${out}  OK
+    ${out}=  Run  govc dvs.portgroup.add -nports 12 -dc=ha-datacenter -dvs=test-ds bridge
+    Should Contain  ${out}  OK
+
+    Log To Console  Add all the hosts to the distributed switch
+    ${out}=  Run  govc dvs.add -dvs=test-ds -pnic=vmnic1 /ha-datacenter/host/cls
+    Should Contain  ${out}  OK
+
+    Log To Console  Enable DRS on the cluster
+    ${out}=  Run  govc cluster.change -drs-enabled /ha-datacenter/host/cls
+    Should Be Empty  ${out}
+
+    Log To Console  Deploy VIC to the VC cluster
+    Set Environment Variable  GOVC_URL  ${vc1-ip}
+    Set Environment Variable  TEST_URL_ARRAY  ${vc1-ip}
+    Set Environment Variable  TEST_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  TEST_PASSWORD  Admin\!23
+    Set Environment Variable  BRIDGE_NETWORK  bridge
+    Set Environment Variable  EXTERNAL_NETWORK  vm-network
+    Set Environment Variable  TEST_DATASTORE  datastore1
+    Set Environment Variable  TEST_RESOURCE  cls
+    Set Environment Variable  TEST_TIMEOUT  30m
+    
+    Install VIC Appliance To Test Server  ${false}  default
```

Comment:
This is what I tested and worked for now, so I would rather stick with this until you can test the changes in your PR and they work.